documentation/API_documentation/Connectivity_API.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+This API makes possible to query the connectivity status of specific user equipment.
+
+## 1\. Introduction
+<img src="resources/DeviceStatus_Connectivity_diagram.png">
+
+## 2\. Quick Start
+
+Sample API invocation is presented in Section 4.4.
+
+
+
+## 3\. Authentication and Authorization
+The Device Status API makes use of the client credentials grant which is applicable for server to server use cases involving trusted partners or clients without any protected user data involved. In this method the API invoker client is registered as a confidential client with an authorization grant type of client_credentials [1].
+
+## 4\. API Documentation
+The API user wants to execute a check of connectivity status for a user equipment. For example API can check the roaming status of device with given identifier.
```

Comment:
The current API version supports the event type 'roaming status' and this can be extended later,......